src/inverseRobot.js

```diff
@@ -24,7 +24,22 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+  const check = Object.values(robot);
+
+  check.sort();
+
+  for (let i = 0; i < check.length; i++) {
+    if (check[i] === check[i + 1]) {
+      return null;
+    }
+  }
```

Comment:
```suggestion
```
You don't need this checks at all. You can simply check if the key is already present in `result` before adding.